README.md

```diff
@@ -37,6 +37,13 @@ $ bundle install
 $ jekyll serve
 ~~~
 
+If that fails you may need to run the Jekyll server with, assuming that
+`bundle install` from the last step was ran.
```

Comment:
Can we remove the conditional and just say, "If that fails you may need to run the command with bundle exec"?